rust/processor/src/processors/account_transactions_processor.rs

```diff
@@ -42,7 +42,7 @@ async fn insert_to_db(
     name: &'static str,
     start_version: u64,
     end_version: u64,
-    account_transactions: Vec<AccountTransaction>,
+    account_transactions: &[AccountTransaction],
```

Comment:
when should we use `&Vec<T>` vs `&[T]`?

Reply:
always use &[T] when we can; [this](https://github.com/rust-lang/rust-clippy/issues/10269#:~:text=Advantage,dereferences%20instead%20of%20one%20(godbolt)) is a great explanation